pkg/operator/ceph/config/config.go

```diff
@@ -71,6 +75,9 @@ var (
 	// VarLogCephDir defines Ceph logging directory. It is made overwriteable only for unit tests where it
 	// may be needed to send data intended for /var/log/ceph to a temporary test dir.
 	VarLogCephDir = "/var/log/ceph"
+
+	// VarCrashCephDir defines Ceph crash reports directory.
+	VarCrashCephDir = path.Join(VarLibCephDir, "crash")
```

Comment:
The name here seems a little strange to me. The rest of the dirs are named directly based on the path, which would make this `VarLibCephCrashDir`.